demos/Demo.ts

```diff
@@ -253,31 +254,31 @@ class Demo {
         }
     }
 
-    public _createExportFileRequest(keyId: string): ExportFileRequest {
+    public _createExportFileRequest(walletId: string): ExportFileRequest {
         return {
             appName: 'Accounts Demos',
-            keyId,
+            walletId,
         } as ExportFileRequest;
     }
 
     public async updateAccounts() {
-        const keys = await this.list();
-        console.log('Accounts in Manager:', keys);
+        const wallets = await this.list();
+        console.log('Accounts in Manager:', wallets);
 
         const $ul = document.querySelector('#accounts');
         let html = '';
 
-        keys.forEach(key => {
-            html += `<li>${key.label}
-                        <button class="export-words" data-keyid="${key.id}">Export Words</button>
-                        <button class="export-file" data-keyid="${key.id}">Export File</button>
-                        <button class="logout" data-keyid="${key.id}">Logout</button>
+        wallets.forEach(wallet => {
+            html += `<li>${wallet.label}
+                        <button class="export-words" data-walletid="${wallet.id}">Export Words</button>
+                        <button class="export-file" data-walletid="${wallet.id}">Export File</button>
+                        <button class="logout" data-walletid="${wallet.id}">Logout</button>
                         <ul>`;
-            key.addresses.forEach((acc, addr) => {
+            wallet.accounts.forEach((acc, addr) => {
                 html += `
                             <li>
                                 <label>
-                                    <input type="radio" name="sign-tx-address" data-address="${addr}" data-keyid="${key.id}">
+                                    <input type="radio" name="sign-tx-address" data-address="${addr}" data-walletid="${wallet.id}">
```

Comment:
I guess in a perfect world it would be data-wallet-id="...". But I'm ok with this world being pretty good. :)